backend/src/main/java/heartbeat/controller/pipeline/PipelineController.java

```diff
@@ -35,6 +36,26 @@ public BuildKiteResponseDTO getBuildKiteInfo(@PathVariable String pipelineType,
 		return buildKiteService.fetchPipelineInfo(pipelineParam);
 	}
 
+	@PostMapping("/{pipelineType}/verify")
+	public ResponseEntity<Void> verifyBuildKiteToken(@PathVariable String pipelineType,
+			@Valid @RequestBody TokenParam tokenParam) {
+		buildKiteService.getBuildKiteVerify(tokenParam.getToken());
+		return ResponseEntity.noContent().build();
+	}
+
+	@PostMapping("/{pipelineType}/info")
+	public ResponseEntity<BuildKiteResponseDTO> fetchBuildKiteInfo(@PathVariable String pipelineType,
```

Comment:
pipelineType to a enum

Reply:
and remove `/info` in the path

Reply:
"/{xxxxType}/info" We have designed this API to all follow this form （pipeline board source）. also the old api hasn't been removed yet, we can distinguish by this.